python/plugins/processing/algs/gdal/rasterize_over.py

```diff
@@ -21,38 +21,59 @@
 __date__ = 'September 2013'
 __copyright__ = '(C) 2013, Alexander Bruy'
 
-import os
+# This will get replaced with a git SHA1 when you do a git archive
+
+__revision__ = '$Format:%H$'
 
-from qgis.core import QgsProcessingUtils
+import os
 
 from qgis.PyQt.QtGui import QIcon
 
+from qgis.core import (QgsRasterFileWriter,
+                       QgsProcessingParameterDefinition,
+                       QgsProcessingParameterFeatureSource,
+                       QgsProcessingParameterField,
+                       QgsProcessingParameterRasterLayer,
+                       QgsProcessingParameterNumber,
+                       QgsProcessingParameterString,
+                       QgsProcessingParameterEnum,
+                       QgsProcessingParameterExtent,
+                       QgsProcessingParameterBoolean,
+                       QgsProcessingParameterRasterDestination)
```

Comment:
Seems this import is not used

Reply:
I removed a few imports, but the one for boolean I think is necessary (for the "add" parameter), am I wrong?